### PR TITLE
Release Day III

### PR DESCRIPTION
**Pull request by ghost**
Release Day III

### CODE REVIEW

File: usage.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Example: `!removeAllRoles`
+
+Removes all roles from the user. _Tabula rasa_, baby.
+
+
+## Administrative commands
+
+Max also includes a couple of special commands just for Officers and above for making announcements to multiple channels.
+
+### !shout
```
Comment (MROwens) [inline usage.md]: Should we remove the !shout and !shoutAll since the functionality will not be in this release?

File: bot/controllers/verify.js [hunk 2]
```diff
@@ -53,11 +62,20 @@ module.exports = () => {
...
                   verified: 1,
-                }).then(util.log).catch(util.error);
+                });
+                // maping guild roles to find the crew role id
+                const guild = message.guild.roles.map(channel => channel);
+                Object.keys(guild).forEach((el) => {
+                  if (guild[el].name === 'crew') {
+                    m.member.addRole(guild[el].id);
+                  } else {
+                    util.log('the moredator has to create the crew role');
```
Comment (buchholz-sean) [inline bot/controllers/verify.js]: Typo: moredator -> moderator; Also, no `output` and `level` params passed to `util.log()`. Recommend `util.log(<message>, el, 2);`

File: usage.md (hunk 1 above)
Comment (buchholz-sean) [reply usage.md]: Agreed. Let's drop this.

File: bot/controllers/verify.js [hunk 3]
```diff
@@ -53,11 +62,20 @@ module.exports = () => {
...
                 // no existing record found
                 models.Member.create({
-                  discorduser: m.author.username,
+                  discorduser: m.author.id,
                   email,
                   uuid: uuidv4(),
                   verified: 1,
-                }).then(util.log).catch(util.error);
+                });
+                // maping guild roles to find the crew role id
```
Comment (buchholz-sean) [inline bot/controllers/verify.js]: Typo: maping -> mapping

File: bot/controllers/verify.js [hunk 4]
```diff
@@ -41,6 +49,7 @@ module.exports = () => {
         if (validDomains.includes(emailDomain)) {
           const codeLength = 6;
           const code = generateCode(codeLength);
+          util.log('code', code);
```
Comment (buchholz-sean) [inline bot/controllers/verify.js]: No debug level param in `util.log()`.

**Review by buchholz-sean — CHANGES_REQUESTED** [review]
Some minor typos. A couple issues with missing params in `util.log()`s. 

Need to change Line 57 value to `600000`

**Review by buchholz-sean — APPROVED** [review]

**Review by MROwens — APPROVED** [review]

**Pull request merged**